src/components/_common/form-details/FormDetails.tsx

```diff
@@ -1,49 +1,59 @@
 import { Heading } from '@navikt/ds-react';
+import classNames from 'classnames';
+
+import { forceArray } from 'utils/arrays';
+import { ParsedHtml } from '../parsed-html/ParsedHtml';
 import { FormDetailsData, Variation } from 'types/content-props/form-details';
 import { FormDetailsVariation } from './FormDetailsVariation';
 
 import styles from './FormDetails.module.scss';
-import classNames from 'classnames';
-import { ParsedHtml } from '../parsed-html/ParsedHtml';
 
 type FormDetailsProps = {
     formDetails: FormDetailsData;
+    displayConfig: {
+        showTitle: boolean;
+        showIngress: boolean;
+        showAddendums: boolean;
+        showApplications: boolean;
+    };
 };
 
-export const FormDetails = ({ formDetails }: FormDetailsProps) => {
-    const { formType } = formDetails;
-
-    const variations = (formDetails.formType[formType._selected]?.variations ||
-        []) as Variation[];
+export const FormDetails = ({
+    formDetails,
+    displayConfig,
+}: FormDetailsProps) => {
+    const { showAddendums, showApplications, showTitle, showIngress } =
+        displayConfig;
 
-    const direction =
-        formDetails.formType._selected === 'addendum'
-            ? 'vertical'
-            : 'horizontal';
+    const variations = forceArray(formDetails.formType).reduce((acc, cur) => {
```

Comment:
Guillotine skal håndtere det så felter som kan være arrays vil alltid være arrays, selv med bare ett element. ForceArray skal egentlig ikke være nødvendig i frontend.

Må oppdatere typen for `formType` til array.

Reply:
Synes liksom jeg har fått inn objekter iblant, men jeg må ha blingset får inn array konsekvent ser jeg. Fjerner!